### ISSUE TITLE
Error for interpolate

### ISSUE THREAD

**Issue by danke1896** [NVIDIA-AI-IOT/torch2trt#297]
My system is ubuntu16.04 gcc 7.4.0 cuda 9.0  tensorrt 5.1.0 pytorch 1.1.0, torchvision 0.3.0 protobuf 3.9 .when  i install torch2trt using 
sudo python setup.py install --plugins
no error occurred.
but when i run python -m torch2trt.test --name=interpolate ,it got nothing. 
It seems that the interpolate layer did not add in converters successfully. Here is the name in test.module_name(), no interpolate.tests.

torch2trt.converters.softmax.test_softmax_module
torch2trt.converters.softmax.test_softmax_module_dim2
torch2trt.converters.split.test_torch_split_1_1
torch2trt.converters.split.test_torch_split_1_1
torch2trt.converters.split.test_torch_split_2_1
torch2trt.converters.split.test_torch_split_2_1
torch2trt.converters.split.test_torch_split_3_1
torch2trt.converters.split.test_torch_split_3_1
torch2trt.converters.split.test_torch_split_3_2
torch2trt.converters.split.test_torch_split_3_2
torch2trt.converters.split.test_tensor_split_3_2
torch2trt.converters.chunk.test_torch_chunk_1_1
torch2trt.converters.chunk.test_torch_chunk_1_1
torch2trt.converters.chunk.test_torch_chunk_2_1
torch2trt.converters.chunk.test_torch_chunk_2_1
torch2trt.converters.chunk.test_torch_chunk_3_1
torch2trt.converters.chunk.test_torch_chunk_3_1
torch2trt.converters.chunk.test_torch_chunk_3_2
torch2trt.converters.chunk.test_torch_chunk_3_2
torch2trt.converters.chunk.test_tensor_chunk_3_2

add The output of ldd -r libtorch2trt.so is as below
linux-vdso.so.1 =>  (0x00007ffc8f9ea000)
	libc10.so => /home/gu/miniconda3/envs/tensorrt/lib/python3.6/site-packages/torch/lib/libc10.so (0x00007f23255b7000)
	libc10_cuda.so => /home/gu/miniconda3/envs/tensorrt/lib/python3.6/site-packages/torch/lib/libc10_cuda.so (0x00007f2325393000)
	libtorch.so.1 => /home/gu/miniconda3/envs/tensorrt/lib/python3.6/site-packages/torch/lib/libtorch.so.1 (0x00007f23240a8000)
	libcudart.so.9.0 => /usr/local/cuda-9.0/lib64/libcudart.so.9.0 (0x00007f2323e3b000)
	libprotobuf.so.20 => /usr/local/lib/libprotobuf.so.20 (0x00007f23235d2000)
	libnvinfer.so.5 => /home/gu/objectDetection/TensorRT/TensorRT-5.1.5.0/lib/libnvinfer.so.5 (0x00007f231cbd8000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f231c7f5000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f231c5dd000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f231c213000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f231bf0a000)
	libgomp.so.1 => /home/gu/miniconda3/envs/tensorrt/lib/python3.6/site-packages/torch/lib/../../../../libgomp.so.1 (0x00007f2325c0e000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f231bced000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f2325a4d000)
	libnvToolsExt.so.1 => /usr/local/cuda-9.0/lib64/libnvToolsExt.so.1 (0x00007f231bae4000)
	libcaffe2.so => /home/gu/miniconda3/envs/tensorrt/lib/python3.6/site-packages/torch/lib/libcaffe2.so (0x00007f23188ae000)
	libcaffe2_gpu.so => /home/gu/miniconda3/envs/tensorrt/lib/python3.6/site-packages/torch/lib/libcaffe2_gpu.so (0x00007f22f605d000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f22f5e59000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f22f5c51000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f22f5a37000)
	libcudnn.so.7 => /usr/local/cuda-9.0/lib64/libcudnn.so.7 (0x00007f22e2cb1000)
	libcublas.so.9.0 => /usr/local/cuda-9.0/lib64/libcublas.so.9.0 (0x00007f22df034000)
	libmkl_intel_lp64.so => /home/gu/miniconda3/envs/tensorrt/lib/python3.6/site-packages/torch/lib/../../../../libmkl_intel_lp64.so (0x00007f22de4c8000)
	libmkl_gnu_thread.so => /home/gu/miniconda3/envs/tensorrt/lib/python3.6/site-packages/torch/lib/../../../../libmkl_gnu_thread.so (0x00007f22dcbef000)
	libmkl_core.so => /home/gu/miniconda3/envs/tensorrt/lib/python3.6/site-packages/torch/lib/../../../../libmkl_core.so (0x00007f22d88cf000)
	libcusparse.so.9.0 => /usr/local/cuda-9.0/lib64/libcusparse.so.9.0 (0x00007f22d5169000)
	libcurand.so.9.0 => /usr/local/cuda-9.0/lib64/libcurand.so.9.0 (0x00007f22d1205000)
	libcufft.so.9.0 => /usr/local/cuda-9.0/lib64/libcufft.so.9.0 (0x00007f22c9164000)
